openghg/retrieve/_search.py

```diff
@@ -254,35 +254,40 @@ def search_footprints(
     """
```

Comment:
The import was moved to the top, it just doesn't show as a change because that's where it was originally